### PR TITLE
fix(deploy): Another artifact of the monitoring package's naming

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/SpinnakerArtifact.java [hunk 1]
```diff
@@ -42,8 +42,8 @@
   IGOR("igor", new String[]{"spinnaker\\.yml", "igor.*\\.yml"}, true),
   ORCA("orca", new String[]{"spinnaker\\.yml", "orca.*\\.yml"}, true),
   ROSCO("rosco", new String[]{"spinnaker\\.yml", "rosco.*\\.yml"}, true),
-  SPINNAKER_MONITORING_THIRD_PARTY("monitoring-third-party", new String[]{}, true),
   SPINNAKER_MONITORING_DAEMON("monitoring-daemon", new String[]{}, true),
+  SPINNAKER_MONITORING_THIRD_PARTY("monitoring-third-party", new String[]{}, false),
```
Comment (jtk54) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/SpinnakerArtifact.java]: Why in this order?

Comment (lwander) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/SpinnakerArtifact.java]: alphabetical

Comment (lwander) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/SpinnakerArtifact.java]: (to make @duftler happy)

Comment (jtk54) [inline halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/SpinnakerArtifact.java]: Heh

**Pull request merged**